---
title: "Projects"
permalink: /projects/
---

Below are a few representative projects. Please reach out if you’d like more details or to work together.

## Single‑cell analyses 
<ul>
  <li>
 Single‑cell RNAseq analyses on *Xenopus* models to study
the gene regulatory networks in neural crest migration during embryonic development. Performed cell type annotations, integration, RNA velocity and pseudotime trajectory analysis, and data visualization with interpretation of the single-cell RNAseq data to trace the differentiation
of neural crest cell types. <a href="https://github.com/nehaprs/DorsalMigration" target="_blank">
    (code)
  </a>
  </li>

  <li>
 Validated two mouse reporter lines for neural crest, and integrated these single‑cell datasets
from two mouse lines to identify marker gene expression patterns. <a href="https://github.com/nehaprs/NeuralCrest_singlecell" target="_blank">
    (code)
  </a>
  </li>
</ul>

## Multi‑omics integration {#multi-omics}
<ul>
  <li>
Applied genomic technologies to predict the substrates and targets of ADAM9 metalloproteinase
and understand its mechanism using colon cancer cell lines.
  </li>

  <li>
 Leverages sequencing methods, including differential expression analysis and machine learning,
to integrate RNAseq, Cellular Proteomics, and Secretomics Datasets.
  </li>

  <li>
 Regression models to predict 176 transcriptional and 1041 post‑transcriptional targets
of ADAM9, including 188 substrates with ranked levels of confidence.
  </li>

  <li>
 Pathway enrichment analysis to validate two downstream pathways.
  </li>

  <li>
 A computational and statistical proteotranscriptomics method is developed, applicable to different
membrane proteins with minimal modifications. <a href="https://github.com/nehaprs/Multi_omics" target="_blank">
    (code)
  </a>
 </li>
 
</ul>

## Omics in drug development {#drug-devl}
<ul>
  <li>
 Paired‑design RNAseq to study the effect of drug treatment on stag2 mutant mouse models. <a href="https://github.com/nehaprs/hippocampus" target="_blank">
    (code)
  </a>
  </li>
  
  <li>
 Drug sensitivity test for novel anti‑osteoarthritis drugs using RNAseq of bovine cartilage. 
  </li>

  <li>
 Differential expression analysis of Parkinsons' drug on mouse models. <a href="https://github.com/nehaprs/parkinsons" target="_blank">
    (code)
  </a>
  </li>
</ul>

## Machine Learning projects
<ul>
  <li>
 Developed models to integrate recursive algorithms with predictive analytics for disease
transmission. Used this to model the spread of COVID‑19 in Delaware.  <a href="https://github.com/nehaprs/LearningCovid" target="_blank">
    (code)
  </a>
  </li>

  <li>
 Developed a machine learning model to predict the co-occurrence of substance use disorders and mood disorders using publicly available datasets. <a href="https://github.com/nehaprs/Personal-Projects/tree/main/Mental%20Health" target="_blank">
    (code)
  </a>
  </li>

  <li>
 Unsupervised integration of biomarker and clinical KPMP data to reveal subtypes of chronic kidney disease. Chronic Kidney Disease(CKD) is characterized by gradual loss of the kidney's ability to filter waste from the blood. In this project, we combine clinical and biomarker data of patients diagnosed with CKD, obtained from [The Kidney Tissue Atlas](https://atlas.kpmp.org/), to classify the various subtypes of the disease. <a href="https://github.com/nehaprs/KPMP_ML" target="_blank">
    (code)
  </a>
  </li>
</ul>
